internal/jennies/golang/rawtypes.go

```diff
@@ -147,6 +142,154 @@ func (jenny RawTypes) formatObject(schema *ast.Schema, def ast.Object) ([]byte,
 			buffer.WriteString("}\n")
 		}
 	}
+}
+
+func (jenny RawTypes) generateConstructor(buffer *strings.Builder, context languages.Context, object ast.Object) {
+	defName := formatObjectName(object.Name)
+	constructorName := "New" + defName
+
+	declareConstructor := func() {
+		jenny.apiRefCollector.RegisterFunction(object.SelfRef.ReferredPkg, common.FunctionReference{
+			Name: constructorName,
+			Comments: []string{
+				fmt.Sprintf("%[1]s creates a new %[2]s object.", constructorName, defName),
+			},
+			Return: "*" + defName,
+		})
+	}
+
+	if object.Type.IsRef() {
+		referredObj, found := context.LocateObjectByRef(*object.Type.Ref)
+		if !found || !referredObj.Type.IsStruct() {
+			return
+		}
+
+		declareConstructor()
+		buffer.WriteString(fmt.Sprintf("// %[1]s creates a new %[2]s object.\n", constructorName, defName))
```

Comment:
Any particular reason to use `"... %[1]s ... %[2]s ..."` in these strings fmts (here and some places below), instead of just `"... %s ... %s ..."` given that we're using same args order everywhere (and not needed to re-refer to any arg)?

Reply:
No real reason, I don't mind changing this if you want :)